ecosystem.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  apps: [
+    {
+      env: {
+        NODE_ENV: 'production',
+      },
+      exp_backoff_restart_delay: 100,
```

Comment:
Given that we're adding this to recover from the SDK-triggered bug, wouldn't it make more sense to have a constant delay rather than the exp backoff?

Reply:
Yeah, that makes sense.